topics/appendix/tools/tools_gradle_intellij_plugin.md

```diff
@@ -0,0 +1,1501 @@
+Takes precedence over [`signPlugin.cliVersion`](#signplugin-task-cliversion).
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### keyStore
+{id="signplugin-task-keystore"}
+
+KeyStore file path.
+Refers to `ks` CLI option.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### keyStorePassword
+{id="signplugin-task-keystorepassword"}
+
+KeyStore password.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### keyStoreKeyAlias
+{id="signplugin-task-keystorekeyalias"}
+
+KeyStore key alias.
+Refers to `ks-key-alias` CLI option.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### keyStoreType
+{id="signplugin-task-keystoretype"}
+
+KeyStore type.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### keyStoreProviderName
+{id="signplugin-task-keystoreprovidername"}
+
+JCA KeyStore Provider name.
+Refers to `ks-provider-name` CLI option.
+
+**Type:** `String`
+
+**Default value:** `null`
+
+
+### inputArchiveFile
+{id="signplugin-task-inputarchivefile"}
+
+Input, unsigned ZIP archive file.
+Refers to `in` CLI option.
+
+Provided by the [`buildPlugin`](#buildplugin-task) task.
+
+### outputArchiveFile
+{id="signplugin-task-outputarchivefile"}
+
+Output, signed ZIP archive file.
+Refers to `out` CLI option.
+
+Predefined with the name of the ZIP archive file with `-signed` name suffix attached.
+
+**Type:** `File`
+
+
+## verifyPlugin Task
+Validates completeness and contents of <path>[plugin.xml](plugin_configuration_file.md)</path> descriptors as well as plugin archive structure.
+
+
+### ignoreFailures
+{id="verifyplugin-task-ignorefailures"}
+
+Specifies whether the build should fail when the verifications performed by this task fail.
+
+**Type**: `Boolean`
+
+**Default value:** `false`
+
+
+### ignoreWarnings
+{id="verifyplugin-task-ignorewarnings"}
+
+Specifies whether the build should fail when the verifications performed by this task emit warnings.
+
+**Type**: `Boolean`
+
+**Default value:** `true`
+
+
+### pluginDir
+{id="verifyplugin-task-plugindir"}
+
+The location of the built plugin file which will be used for verification.
+
+**Type**: `File`
+
+**Default value:** `${prepareSandboxTask.destinationDir}/${prepareSandboxTask.pluginName}`
+
+
+
+## Build Features
+With the Gradle IntelliJ Plugin releases, new features are introduced that require additional research, collecting more feedback from developers, or should be enabled or disabled under particular conditions.
+Build Features are an implementation of the feature flags concept and let you control some behaviors of the Gradle IntelliJ Plugin.
+To enable or disable a particular feature, add the Project property to the <path>gradle.properties</path> file, like:
+
+```properties
+org.jetbrains.intellij.buildFeature.buildFeatureName=false
+```
+
+### selfUpdateCheck
+{id="build-features-selfupdatecheck"}
+
+Check if the currently used Gradle IntelliJ Plugin is outdated.
+
+**Default value:** `true`
+
+
+
+## Frequently Asked Questions
```

Comment:
I'm thinking we should extract FAQ to subpage of this so it;s clearly a Reference only
the page is already way too long and we'll certainly add more FAQ entries over time